utils/firestore.js

```diff
@@ -996,61 +998,34 @@ const searchSpecimen = async (masterSpecimenId, siteCode) => {
 }
 
 const searchShipments = async (siteCode) => {
-    const snapshot = await db.collection('biospecimen').where('827220437', '==', siteCode).get();
+    const { reportMissingTube, submitShipmentFlag, healthCareProvider} = fieldMapping;
+    const snapshot = await db.collection('biospecimen').where(healthCareProvider, '==', siteCode).get();
+
     if(snapshot.size !== 0){
-        //
         return snapshot.docs.filter(document => {
```

Comment:
This filter call is bulky.

Reply:
Thanks this is now replaced with a `for... loop`